targets/mgwire/targets.go

```diff
@@ -1,11 +1,10 @@
 package mgwire
 
 import (
-	"github.com/magefile/mage/sh"
-	"go.einride.tech/mage-tools/mglog"
+	"go.einride.tech/mage-tools/mgtool"
+	"os/exec"
 )
 
-func WireGenerate(path string) error {
-	mglog.Logger("wire-generate").Info("generating initializers...")
-	return sh.RunV("go", "run", "-mod=mod", "github.com/google/wire/cmd/wire", "gen", path)
+func WireGenerate(path string) *exec.Cmd {
```

Comment:
IMO this might be better formulated as

```go
func Command(args ...string) *exec.Cmd {
    args = append(args, "go", "run", "-mod=mod", "github.com/google/wire/cmd/wire")
    return mgtool.Command(args)
}
```